plugins/exporter.koplugin/target/xmnote.lua

```diff
@@ -0,0 +1,166 @@
+local InputDialog = require("ui/widget/inputdialog")
+local UIManager = require("ui/uimanager")
+local InfoMessage = require("ui/widget/infomessage")
+local BD = require("ui/bidi")
+local DataStorage = require("datastorage")
+local http = require("socket.http")
+local json = require("json")
+local logger = require("logger")
+local ltn12 = require("ltn12")
+local socket = require("socket")
+local socketutil = require("socketutil")
+local util = require("ffi/util")
+local T = util.template
+local _ = require("gettext")
+
+-- xmnote exporter
+local XMNoteExporter = require("base"):new {
+    name = "xmnote",
+    is_remote = true,
+    server_port = 8080
```

Comment:
That's not dynamic?

Reply:
The port number is fixed as 8080 in the application(XMNote), which means its value does not change over time.